planet.rb

```diff
@@ -0,0 +1,25 @@
+require 'faker'
+def autofactoid(planet_name)
```

Comment:
Later on, I would suggest making this a private helper method, which is a concept we'll get to later.